Source/Core/loadCRN.js

```diff
@@ -0,0 +1,84 @@
+/*global define*/
+define([
+    './CompressedTextureBuffer',
+    './defined',
+    './DeveloperError',
+    './loadArrayBuffer',
+    './TaskProcessor',
+    '../ThirdParty/when'
+], function(
+    CompressedTextureBuffer,
+    defined,
+    DeveloperError,
+    loadArrayBuffer,
+    TaskProcessor,
+    when) {
+    'use strict';
+
+    var transcodeTaskProcessor = new TaskProcessor('transcodeCRNToDXT', Number.POSITIVE_INFINITY);
```

Comment:
This is the only option because anytime we load an image we expect a promise that will resolve to an image. Otherwise, we have to change that behavior so when `undefined` is returned the task is rescheduled. We use this option for geometry web workers as well.